docker-compose.yml

```
@@ -164,4 +190,4 @@ services:
 
 networks:
   mynetwork:
-    driver: bridge
+    driver: bridge
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)